sources/engine/Stride.Particles/Components/ParticleSystemSimulationProcessor.cs

```diff
@@ -88,7 +88,7 @@ private void UpdateParticleSystem(ParticleSystemComponentState state, float delt
                 particleSystem.UniformScale = xScale;
             }
 
-            particleSystem.Update(deltaTime * speed);
+            particleSystem.Update(deltaTime * (float) GameTime.Factor * speed);
```

Comment:
Cast/convert to float the result of the three mult instead of ``factor`` itself, might as well use full precision since float mult isn't significantly faster than double mult on current architecture.